Source/RapyutaSimulationPlugins/Public/Core/RRMeshData.h

```diff
@@ -62,6 +63,53 @@ struct RAPYUTASIMULATIONPLUGINS_API FRRBoneProperty
     UPROPERTY(VisibleAnywhere)
     float Mass = 0.f;
 
+    FRRRobotLinkProperty ConvertToLinkProperty() const
```

Comment:
what is the difference between FRRRobotLinkProperty and FRRBoneProperty?

Reply:
So as we know, bone is the concept used in UE skeleton, skeletal mesh
https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/AnimationCore/Public/BoneIndices.h
Robot link is mostly a data representation of robot description format like URDF/SDF
* When we build skeletal mesh, skeleton, we need to construct the bone structure from robot link info loaded from URDF/SDF
* However, if there is no URDF/SDF as in case of loading from a single CAD (like FBX), this conversion util is used to auto-construct a template robot model info from bone info, loaded from FBX.